schema/mysql.sql

```diff
@@ -2446,4 +2446,4 @@ CREATE TABLE branched_icinga_dependency (
 
 INSERT INTO director_schema_migration
   (schema_version, migration_time)
-  VALUES (187, NOW());
+  VALUES (189, NOW());
```

Comment:
Shouldn't 188 be next?

Reply:
I wanted to match it with the postgres migration version. And the postgres migration version was not updated to 188 previously even though there was a migration script for the version 188. So I thought it was right to use 189 instead of 188.

Reply:
Okay. There was a gap between 78-82 anyway with MySQL.

Reply:
#2946 adds version 188.